otx/algorithms/action/adapters/mmaction/models/recognizers/recognizer3d.py

```diff
@@ -0,0 +1,176 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+# Copyright (C) 2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import torch
+from torch import nn
+
+from mmaction.models.builder import RECOGNIZERS
+from mmaction.models.recognizers.base import BaseRecognizer
+import functools
+from collections import OrderedDict
+
+
+@RECOGNIZERS.register_module(force=True)
```

Comment:
Is the `RECOGNIZERS` used in only `Action task` ? If yes, how about changing this name ? `RECOGNIZERS` is a kind of general term.

Reply:
I agree with you, but I followed mmaction2 convention and seems like there is no conflict between mmxxx frameworks. And.. I think using same name convention might be useful to support public backbone for action tasks.

Reply:
I think this is okay